### ISSUE TITLE
Add configuration for increase max file size

### ISSUE THREAD

**Issue by NitramDom** [n1crack/vuefinder#5]
Could you add an config parameter to setup max file size ?

**Comment by n1crack** [comment]
Yes, It is possible.I also think upload type filter also needed. 

Btw, max file size should be permitted by the server side script. In order to pass php's limit, I was thinking using chunks for uploads. But it requires some works.

**Comment by NitramDom** [comment]
For my personal use, I only need to increase the file size on front side.

**Comment by n1crack** [comment]
it is set to 10mb right? If so, it will be configurable when I get home.

**Comment by n1crack** [comment]
it is done. example below

```vue

<vue-finder dark locale='en' id='local' url="/ajaxUrl" max-file-size="50mb"></vue-finder>

```

**Comment by NitramDom** [comment]
Did you release-it ?

**Comment by n1crack** [comment]
Yes. `npm update`
